vnet/spec/vnet/endpoints/1.0/networks_spec.rb

```diff
@@ -38,10 +38,7 @@ def app
   describe "PUT /:uuid" do
     accepted_params = {
       :display_name => "our new name for the test network",
-      :ipv4_network => "10.0.0.2",
-      :ipv4_prefix => 8,
       :domain_name => "new.vdc.test.domain",
-      :network_mode => "physical",
```

Comment:
Same again. As long as these parameters are still accepted by the webapi, we need to test them.